transport/http/handler.go

```diff
@@ -228,3 +236,18 @@ func (rw *responseWriter) Close(httpStatusCode int) {
 		bufferpool.Put(rw.buffer)
 	}
 }
+
+func getContentType(encoding transport.Encoding) string {
+	switch encoding {
+	case "json":
+		return "application/json"
+	case "raw":
+		return "application/octet-stream"
+	case "thrift":
+		return "application/x-thrift"
```

Comment:
Guess what we learned on Monday. The IANA is `application/vnd.apache.thrift.binary` (unless it’s enveloped).